src/harness.js

```diff
@@ -234,11 +235,41 @@ class Harness {
       /* this.log respects verbose option, use logFormErrors here */
       console.error(`Error encountered while filling form:`, JSON.stringify(fillResult.errors, null, 2));
     }
+    return fillResult;
+  }
 
+  /**
+   * Loads and fills a contact form,
+   *
+   * @param {string} contactType Type of contact that should be created
+   * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+   */
+  async fillContactForm(contactType, ...answers) {
+    const fillResult = await this._fillContactForm(contactType, 'create', ...answers);
     this.pushMockedDoc(...fillResult.contacts);
     return fillResult;
   }
 
+  /**
+   * Loads and fills a contact edit form,
+   *
+   * @param {string} contactType Type of contact that should be created
+   * @param  {...string[]} answers Provide an array for the answers given on each page. See fillForm for more details.
+   */
+  async fillContactEditForm(contactType, ...answers) {
+    const assignValues = (target, props) =>
```

Comment:
this is hard to read and understand - single letter variable names, very compact statements, assignment chained with conditions. i'd recommend trying to re-use existing library functions for this. is this useful?https://lodash.com/docs/#assignInWith

Reply:
Yes that works as well